base/libgit2/rebase.jl

```diff
@@ -69,11 +77,27 @@ function commit(rb::GitRebase, sig::GitSignature)
     return oid_ptr[]
 end
 
+"""
+    abort(rb::GitRebase) -> Csize_t
+
+Cancels the in-progress rebase, undoing all changes made so far and returning
+the parent repository of `rb` and its working directory to their state before
+the rebase was initiated. Returns `0` if the abort is successful, `GIT_ENOTFOUND`
+if no rebase is in progress (for example, if the rebase had completed), and `-1`
+for other errors.
+"""
 function abort(rb::GitRebase)
     return ccall((:git_rebase_abort, :libgit2), Csize_t,
                       (Ptr{Void},), rb.ptr)
 end
 
+"""
+    finish(rb::GitRebase, sig::GitSignature) -> Csize_t
+
+Complete the rebase described by `rb`. `sig` is an optional [`GitSignature`](@ref)
```

Comment:
I went on a bit of a dive through the `libgit2` docs and I *suspect* if you pass `NULL` as a pointer it'll use the default sig from config ... but let's remove this language just in case.